packages/pwa-kit-react-sdk/src/ssr/universal/components/storefront-preview/index.jsx

```diff
@@ -16,9 +16,10 @@ import {useHistory} from 'react-router-dom'
  * @param {boolean} enabled - flag to turn on/off Storefront Preview feature
  * @param  {function(): string | Promise<string>} getToken - A method that returns the access token for the current user
  */
-export const StorefrontPreview = ({enabled = true, getToken}) => {
+export const StorefrontPreview = ({children, enabled = true, getToken}) => {
```

Comment:
Since we're already `export default StorefrontPreview`, then we don't need to export this one too.
```suggestion
const StorefrontPreview = ({children, enabled = true, getToken}) => {
```